java/Makefile

```diff
@@ -745,17 +802,22 @@ else
 	@echo "Using Error Prone version ${error-prone-version}"
 ## MutablePublicArray is off because it suggests a dependency on Google Guava.
 	javac -cp ${DAIKON_CLASSPATH} \
+	$(extra-error-prone-arg-2) \
 	  ${JAVAC_ANNOTATION_PROCESSOR_ARGS} \
 	  -XDcompilePolicy=simple -processorpath ${ERRORPRONE_PROCESSOR_PATH} \
 	  '-Xplugin:ErrorProne -Xep:ReferenceEquality:OFF -Xep:StringSplitter:OFF -Xep:AnnotateFormatMethod:OFF -Xep:MutablePublicArray:OFF -Xep:DoNotCallSuggester:OFF -Xep:InlineMeSuggester:OFF -Xep:CanIgnoreReturnValueSuggester:OFF -Xep:ExtendsObject:OFF -Xep:PatternMatchingInstanceof:OFF' \
-	  -Xmaxwarns 100000 -Werror \
-	  ${extra-error-prone-args} \
+	  -Xmaxwarns 100000 \
+	  ${extra-error-prone-arg-1} \
 	  ${JAVA_FILES_FOR_STYLE}
 endif
 
+# THIS IS BROKEN - IS IT EVER USED?
 # Create an `error-prone.patch` file.
 errorprone-replacements:
 	javac -cp ${DAIKON_CLASSPATH} \
+ifeq ($(shell test ${JAVA_RELEASE_NUMBER} -gt 23; echo $$?),0)
+	  -source 24 -target 24 --enable-preview \
```

Comment:
Is this the same as `${extra-error-prone-args-2}`?  That is, could you use that variable?